### DISCUSSION TITLE
Is there a way to have the code copied to RAM (PICO_COPY_TO_RAM) when compiling from Arduino IDE

### DISCUSSION THREAD

**Discussion by jeenode** [earlephilhower/arduino-pico#2343] [General]
Is there a way to have the code copied to RAM like cmake does with PICO_COPY_TO_RAM when compiling via Arduino IDE?

**Comment by earlephilhower** [comment]
No, that requires a different linker script and is not possible w/the IDE.  Individual functions can be specified to be copied to RAM before use (for speed or because the flash is in write/erase operation) can use the SDK's `__no_inline_not_in_flash_func` macro.

**Reply by jeenode** [reply]
Thanks!
